docs/README.md

```diff
@@ -104,63 +104,110 @@ We will review your changes, add any feedback and once everything looks good mer
+
+   - `content/latest/api/ysql/syntax_resources/commands/copy_from,copy_to,copy_option.diagram.md`
+   - `content/latest/api/ysql/syntax_resources/commands/copy_from,copy_to,copy_option.grammar.md`
+
+1. Create the docs page file:
+
+   - `content/latest/api/ysql/commands/<name>.md`
+
+   Use the existing files in that directory as examples.  For `<name>`, follow the naming convention
+   exhibited by the other files.  For YSQL, note the usage of the template `includeMarkdown` macro.
+
+   _Example: for the YSQL `COPY` command, the created file is the following:_
+
+   - `content/latest/api/ysql/commands/cmd_copy.md`
+
+1. Update the index page files:
+
+   - `content/latest/api/ysql/_index.md`
+   - `content/latest/api/ysql/commands/_index.md`
+
+1. [Run the diagram generator](#generate-api-syntax-diagrams).
+
+   Ensure that no `WARNING`s are thrown related to the rules that you added.
+
+1. Check the new docs page and index pages to make sure that there are no broken links.
+
+### Edit rules in a docs page
+
+You may need to add or edit grammar rules in an existing docs page.
+
+1. Add or edit rules in the source grammar file:
+
+   - `content/latest/api/ysql/syntax_resources/ysql_grammar.ebnf`
+
+1. Rename the diagram and grammar files if needed:
+
+   - `content/latest/api/ysql/syntax_resources/commands/<oldrules>.diagram.md` to
+     `content/latest/api/ysql/syntax_resources/commands/<newrules>.diagram.md`
+   - `content/latest/api/ysql/syntax_resources/commands/<oldrules>.grammar.md` to
+     `content/latest/api/ysql/syntax_resources/commands/<newrules>.grammar.md`
 
-5. Run the diagram generator using the following command:
+1. Edit the contents of the docs page file:
 
-    ```bash
-    java -jar rrdiagram.jar <input-file.ebnf> <output-folder>
-    ```
+   - `content/latest/api/ysql/commands/<name>.md`
 
-    `<output-folder>` should end in `syntax_resources`, not `commands`, to get helpful `WARNING`s.
+1. [Run the diagram generator](#generate-api-syntax-diagrams).
 
-    _Example: To generate the syntax diagrams for the YSQL API, run the following command:_
-    ```bash
-    java -jar rrdiagram.jar content/latest/api/ysql/syntax_resources/ysql_grammar.ebnf content/latest/api/ysql/syntax_resources/
-    ```
+   Ensure that no `WARNING`s are thrown related to the rules that you added or edited.
 
-    All of the Markdown (`.md`) files in the `./ysql/syntax_resources/commands` directory will be generated as needed.
+1. Check the modified docs page to make sure that there are no broken links.
 
-    _Note: To see help, run `java -jar rrdiagram.jar` (without arguments)._
+### Notes
 
-6. Check that the output file looks fine.
-    You may need to save the main Markdown (`.md`) file (for example `commands/cmd_copy.md`) to force the page to be rerendered.
+- You may need to save the docs page file (for example, `commands/cmd_copy.md`) to force the page to
```

Comment:
Change to "To force the page to
  be re-rendered, you may need to save the docs page file (for example, `commands/cmd_copy.md`)."